pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -271,6 +294,52 @@ properties:
       default: {"id": "1", "name": "John Doe"}
 ----
 
+[[dataweave-expressions]]
+=== DataWeave Expressions
+
+The `dataweave` format for `string` type parameters specifies the input parameter as a DataWeave expression.
+
+The `dataweave` variable supports the following expressions bindings:
+
+[source,yaml]
+----
+<configuration-parameter>:
+ type: string
+ format: dataweave
+ bindings:
+   payload:
+     mimeType: text
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `payload`: The body of the imcoming request. Define the `mimeType` as either `text` or `json` to set how to read the payload. By default, `mimeType` is set to `text`. To restrict the use of the `payload` variable, you must exclude the payload variable, for example:
++
+[source,yaml]
+----
+bindings:
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `attributes`: Attributes of the incoming request such as headers. Set `attributes` as `true` to enable the policy to read the attributes of the incoming request. By default, attributes is set to `true`.
+* `authentication`: Authentication information of the incoming request. Set `authentication` to `true`to enable the policy to read authenticaton details of the incoming request. By default, `authentication is set to false.
+* `vars`: Specifies the names of the variables that are available for the policy to use. By default, `vars` is empty. If the user inputs variables that are not defined in the schema, they evaluate to null.
```

Comment:
null?

Reply:
Is the question if null should be in monospace or if null is the correct usage?

Reply:
monospace